commons/connector/health/aggregation/health_body_measurements.avsc

```diff
@@ -0,0 +1,65 @@
+{
+  "namespace": "org.radarcns.active.healthkit",
+  "name": "HealthKitBodyMeasurements",
+  "type": ["null", "record"],
+  "default": null,
+  "doc": "height, weight, bmi, fat_percentage, temperature, waist_circumference, gender, date_of_birth",
```

Comment:
The fields can be read from the field list. Please add a description of this record type instead.

Reply:
Hi @blootsvoets  What do you mean by the field list, can you have an example what I should add here ?